parsedmarc/__init__.py

```diff
@@ -1428,12 +1435,44 @@ def get_dmarc_reports_from_mailbox(connection: MailboxConnection,
         connection.create_folder(smtp_tls_reports_folder)
         connection.create_folder(invalid_reports_folder)
 
-    messages = connection.fetch_messages(reports_folder, batch_size=batch_size)
+    if since:
+        _since = 1440  # default one day
+        if re.match(r'\d{1,2}[mhd]$', since):
```

Comment:
Why restrict to two digits ? (eg: 365 days)
I suggest to use \d+ as the newt regex